scripts/grading/grade_submission.py

```
@@ -1470,10 +1664,60 @@ def run_grading(
+                rubric_criteria = [
+                    {
+                        "criterion_id": f"C{i + 1}",
+                        "criterion_name": c.get("criterion_name", f"Criterion {i + 1}"),
+                        "max_points": float(c.get("max_points", 0)),
+                        "checklist_items": c.get("checklist_items", []),
+                    }
+                    for i, c in enumerate(raw_criteria)
+                ]
+                print(f"JSON rubric loaded: {len(rubric_criteria)} criteria, "
+                      f"{sum(c['max_points'] for c in rubric_criteria):.0f} total pts")
+        except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)